frontend/admin/src/js/components/dashboard/LatestRequestsTable.tsx

```diff
@@ -62,7 +62,12 @@ const LatestRequestsTable: React.FC<LatestRequestsTableProps> = ({ data }) => {
       accessor: ({ id, fullName }) =>
         tableViewItemButtonAccessor(
           paths.searchRequestView(id),
-          "request",
+          intl.formatMessage({
+            defaultMessage: "request",
+            id: "dKtPxj",
+            description:
+              "Text displayed on the search request table view buttons.",
```

Comment:
Consider changing this to "Text displayed after View text for Request table view action"